pages/faq.tsx

```diff
@@ -20,59 +20,120 @@ const questions: ReadonlyArray<{
   id: string
 }> = [
   {
-    question: `How long will the incentivized testnet run for?`,
+    question: `How long will the Phase 2 incentivized testnet run for?`,
     id: `testnet-duration`,
-    answer: `The testnet will run until the Iron Fish node is stable, feature-complete, and ready for mainnet 🎉`,
+    answer: `The testnet will run from May 10, 2022 until the Iron Fish node is stable, feature-complete, and ready for mainnet 🎉`,
   },
   {
-    question: `Can I participate in the Incentivized Testnet if I’m in the US?`,
-    id: `us-participants`,
+    question: `I participated in Phase 1. What’s different?`,
+    id: `phase-1-changes`,
     answer: (
       <div>
-        Yes. Please see{' '}
-        <span className="underline">
-          <Link href="/about#guidelines">Testnet Guidelines</Link>
-        </span>{' '}
-        for more details.
+        <li>
+          We are now rewarding points{' '}
+          <span className="underline">
+            {' '}
+            <Link href="/about"> for different actions</Link>
+          </span>{' '}
+          - you get points for hosting a node and making transactions, and you
+          no longer get points directly for mining tokens. The other categories
+          for points remain the same.
+        </li>
+        <li>
+          Additionally, we have added a number of commands to the CLI to make
+          participating in the testnet simpler - these will help you set up your
+          graffiti and telemetry. Run <code>ironfish testnet</code> to set up
+          your node to be ready for testnet. Check out our{' '}
+          <span className="underline">
+            <Link href="https://ironfish.network/docs/onboarding/iron-fish-tutorial">
+              getting started
+            </Link>
+          </span>{' '}
+          for more details!
+        </li>
+      </div>
+    ),
+  },
+  {
+    question: `Why the point changes?`,
+    id: `why-changes`,
+    answer: (
+      <div>
+        <li>
+          De-incentivizing direct mining gives you the opportunity to join a
+          mining pool and earn proportional shares of blocks mined by a group of
+          miners. No more need for a crazy mining rig - just join up with a
+          number of other miners and get a proportional share of the new tokens
+          mined!
+        </li>
+        <li>
+          And if you do have that amazing mining rig, don’t worry - you will
+          still be able to earn lots of points by making large transactions with
+          the Iron Bank.
+        </li>
       </div>
     ),
   },
   {
-    question: `How do I get points for mining blocks? `,
-    id: `how-do-i-get-points`,
+    question: `How are you rewarding points?`,
+    id: `how-points`,
```

Comment:
Changing ids breaks links, though relative to phase 2 being separate, this is probably ok